hw3/js/hw3-5.js

```diff
@@ -0,0 +1,40 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const operation = prompt('Enter operation (add, sub, mult, div)', '');
+    const isAdd = operation === 'add';
+    const isSub = operation === 'sub';
+    const isMult = operation === 'mult';
+    const isDiv = operation === 'div';
+
+    const firstNumber = prompt('Enter the first number', '');
+    const firstNumberTypeofNumber = Number(firstNumber);
+
+    const secondNumber = prompt('Enter the second number', '');
+    const secondNumberTypeofNumber = Number(secondNumber);
+
+    let operationSymbol;
+    let result;
+
+    if (isAdd) {
+        operationSymbol = '+';
+        result = firstNumberTypeofNumber + secondNumberTypeofNumber;
+    } else if (isSub) {
+        operationSymbol = '-';
+        result = firstNumberTypeofNumber - secondNumberTypeofNumber;
+    } else if (isMult) {
+        operationSymbol = '*';
+        result = firstNumberTypeofNumber * secondNumberTypeofNumber;
+    } else if (isDiv) {
+        operationSymbol = '/';
+        result = firstNumberTypeofNumber / secondNumberTypeofNumber;
+    }
+
+    const goodFirstNumber = typeof firstNumberTypeofNumber === 'number' && !isNaN(firstNumberTypeofNumber);
+    const goodSecondNumber = typeof secondNumberTypeofNumber === 'number' && !isNaN(secondNumberTypeofNumber);
+    const goodResult = typeof result === 'number' && !isNaN(result) && result !== Infinity;
```

Comment:
Why do you need prev 3 checks and variables?

Reply:
To avoid invalid data entry, for example the user will enter letters instead of numbers and then I check it before showing results.